platform/sync-installer/sync-installer-impl/src/main/java/org/codice/ddf/sync/installer/impl/SynchronizedInstallerImpl.java

```diff
@@ -448,7 +450,22 @@ void wait(
     }
   }
 
-  private Set<Bundle> getUnavailableBundles(Set<String> toCheck) {
+  private void printBundleDiags(Collection<Bundle> toPrint) {
+    StringBuilder sb = new StringBuilder();
+    toPrint.forEach(bundle -> appendBundleDiag(bundle, sb));
+    LOGGER.info("Printing unsatisfied bundle requirements: {}", sb);
```

Comment:
Should this be WARN or ERROR to be more visible to the administrator that something went wrong?